docs(submitting): add more info around the RFC process (#1716)
spinnaker/spinnaker.github.io: community/contributing/submitting.md

channel](https://spinnakerteam.slack.com/messages/C0DPVDMQE/) in slack and
discuss your ideas before implementing them.

It's possible that we'll encourage you to write an RFC if your change is large
or impactful enough. There is an [RFC
template](https://github.com/spinnaker/governance/blob/master/rfc/.template.md)

> you should use for this. There are also several [already-submitted

Reviewer: ```suggestion
to use for this. There are also several [already-submitted
```

RFCs](https://github.com/spinnaker/governance/tree/master/rfc) you can use for
reference.

> Once your RFC is ready, add it to [the `spinnaker/governance/rfc`

Reviewer: ```suggestion
When your RFC is ready, add it to [the `spinnaker/governance/rfc`
```

directory](https://github.com/spinnaker/governance/tree/master/rfc) by sending a

> pull request. It will then be reviewed by the community.

Reviewer: ```suggestion
pull request. Members of the community will review it.
```
